src/scheduler/pbs_sched.c

```diff
@@ -567,6 +567,15 @@ server_command(char **jid)
 		return SCH_ERROR;
 	}
 
+	if (set_nodelay(new_socket) == -1) {
+#ifdef WIN32
```

Comment:
why are we adding windows specific code in pbs_sched.c when there is pbs_sched_win.c for windows already present. I think you don't need it here.

Reply:
Yeah, you are right. I will remove it.